packages/@okta/vuepress-site/docs/guides/set-up-token-exchange/main/index.md

```diff
@@ -0,0 +1,345 @@
+1. Click **Save** and at the top of the page click **Back to Applications**.
+
+> **Note:** We recommend that native applications use the Authorization Code with PKCE authentication flow. See [Implement authorization by grant type](/docs/guides/implement-grant-type/authcodepkce/main/#create-the-proof-key-for-code-exchange) for more information on creating the Proof Key for Code Exchange (PKCE) for your native application.
+
+### Create a service app
+
+In token exchange use cases, an API microservice can act both as a resource server and a client. For this example, the native app gets an access token to make API requests to API1. API1 acts as the resource server. When API1 needs to talk to API2 on behalf of the user, the API1 becomes the OAuth client.
+
+Create a service app that represents API1.
+
+1. On the **Applications** page, click **Create App Integration**.
+1. Select **API Services**, and then click **Next**.
+1. Name your application. For this example, enter **API1**, and then click **Save**.
+1. In the **General Settings** section of the **General** tab, click **Edit** and select **Token Exchange** as the grant type for your application.
+1. Click **Save**.
+1. Copy the client ID in the **Client Credentials** section, and then copy the client secret in the **CLIENT SECRETS** section.
+1. [Base64 encode](/docs/guides/implement-grant-type/clientcreds/main/#flow-specifics) the client ID and client secret for use in the token exchange request from API1 to API2.
+
+### Update the custom authorization servers
+
+You need to complete a few tasks to update the custom authorization servers that you plan to use for this example.
+
+#### Create custom scopes
+
+Add some custom scopes to both custom authorization servers to request during the token exchange calls.
+
+> **Note:** You can't include the `offline_access` scope or any of the OpenID Connect scopes when using a service app. This means that you can't request refresh tokens or ID tokens in a service app-initiated token exchange flow.
+
+1. From the Admin Console, select **Security** > **API**, and then select one of the authorization servers that you want to use.
+1. Select the **Scopes** tab, and then click **Add Scope**.
+1. In the dialog that appears, enter a scope name (**api:access:read**).
+1. Click Create.
+1. Repeat steps 1-4 and create a second custom scope called **api:access:write**.
+1. Repeat steps 1-5 for the second authorization server that you want to use.
+
+#### Update audience
+
+Just for this example, update the audience of the authorization servers to help with understanding the token exchange flow when you view the responses.
+
+1. On the **API** page, select the edit (pencil) icon to the right of one of the authorization servers.
+1. In the **Settings** section, click **Edit**.
+1. Change the **Audience** to **com.api.atko** and click **Save**.
+1. Repeat steps 1-3 for the second authorization server.
```

Comment:
I am thinking, if this step will create the confusion that the audience of AS1 has to be the same as the audience of AS2. 
Maybe we can have different audiences for AS1 and AS2 to eliminate this confusion.